playbook.yml

```diff
@@ -1,20 +1,24 @@
 site:
   title: Vahera Documentation
   url: https://vahera.com/docs
-  start_page: orchestrator::overview.adoc
+  start_page: en::overview.adoc
 
 content:
   sources:
     - url: https://github.com/Vahera/godot-orchestrator.git
       branches: 
         - 'main'
-      start_path: docs
+      start_path: docs/en
+    - url: https://github.com/Vahera/godot-orchestrator.git
+      branches: 
+        - 'main'
+      start_path: docs/ru
```

Comment:
So I don't care for how this is rendered, so lets take a step back and lets break the `playbook.yml` into two separate files, `playbook_en.yml` and `playbook_ru.yml`.  

For `playbook_en.yml` there will be a single content source defined as follows:
```yml
context:
  sources:
    - url: https://github.com/Vahera/godot-orchestrator.git
      branches:
        - 'main'
      start_page: docs/en
```

Then for `playbook_ru.yml`:
```yml
context:
  sources:
    - url: https://github.com/Vahera/godot-orchestrator.git
      branches:
        - 'main'
      start_page: docs/ru
```

What we'll then do is use a different output directory for each playbook:

In `playbook_en.yml`:
```yml
output:
  dir: ./docs-build/en
```
and for `playbook_ru.yml`:
```yaml
output:
  dir: ./docs-build/ru
```

This will allow for nice URLs:
https://vahera.com/docs/en/orchestrator/2.0.0/overview.html
https://vahera.com/docs/ru/orchestrator/2.0.0/overview.html

We'll modify the custom template UI layout with a drop-down widget so that the brand supports switching between the two languages and this avoids tainting component and project names for future expansion.

Reply:
This means we'll need to run Antora n times, once for each language, but I think that's a worthy trade-off to avoid injecting something into Antora that it isn't currently designed to support. I know Dan, the project lead on Antora, so I'll reach out and find out what his expected lead time is on this multi-lingual feature.

Reply:
@VladimirLevadnij let me know if you have any questions on my feedback.

Reply:
@Naros I made the changes you wrote about. But, except for one case. You suggested changing `start_page: en::overview.adoc` to `start_page: overview.adoc` in the `playbook.yml` file (which is now divided into two files `playbook_en.yml` and `playbook_ru.yml`), but I got an error with this change "Missing component name in start page for site: overview.adoc", so I made this line as it was originally `start_page: orchestrator::overview.adoc`.

In this version, as I see it, everything works. I was able to create local documentation in English and documentation in Russian (the documentation is not translated yet).

All new commits are already in this pull request.